qemu/tests/nested_block_resize_l1.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import time
+
+from virttest import error_context
+from provider import message_queuing
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+    QEMU nested block resize test of L1
+
+    1) Boot the vm attach the disk from L1
+    2) Run io on the data disk then notify L1
+    3) Execute block_resize for data disk image on host.
+    4) L1 do block_resize and check status of L1 and L2.
```

Comment:
This step is executed on host, and already included in qemu/tests/nested_block_resize.py.

Reply:
updated